apps/condo/domains/onboarding/components/WelcomePopup.tsx

```diff
@@ -0,0 +1,192 @@
+import { Button } from '@condo/domains/common/components/Button'
+import { useIntl } from '@core/next/intl'
+
+const modalCss = css`
+  .ant-modal-content {
+    padding: 0;
+  }
+  
+  .ant-modal-header {
+    border-bottom: none;
+    padding: 40px;
+    
+    .ant-typography {
+      font-weight: 700;
+      font-size: 20px;
+      line-height: 28px;
+    }
+  }
+  
+  .ant-modal-body {
+    padding: 0 40px 20px 40px;
+  }
+
+  .ant-modal-footer {
+    padding: 0;
+    
+    .ant-btn {
+      border: 1px solid ${colors.inputBorderGrey};
+      padding: 4px 15px;
+    }
+  }
+`
+
+const crossIconStyles = css`
+  position: relative;
+  bottom: 8px;
+  right: 3px;
+`
+
+const ModalHeader = ({ handleModalClose }) => {
+    const intl = useIntl()
+    const TitleMessage = intl.formatMessage({ id: 'WelcomePopup.title' })
+
+    return (
+        <Row justify={'space-between'}>
+            <Col>
+                <Typography.Title level={3}>{TitleMessage}</Typography.Title>
+            </Col>
+            <Col>
+                <CrossIcon onClick={handleModalClose} css={crossIconStyles}/>
+            </Col>
+        </Row>
+    )
+}
+
+const MODAL_ROW_STYLES = { padding: '20px 40px' }
+
+const ModalFooter = ({ step, setStep, handleModalClose }) => {
+    const intl = useIntl()
+    const OkMessage = intl.formatMessage({ id: 'OK' })
+
+    const handleNextStepButtonClick = useCallback(() => setStep(currentStep => currentStep + 1), [setStep])
+    const handlePrevStepButtonClick = useCallback(() => setStep(currentStep => currentStep - 1), [setStep])
+
+    return (
+        <Row style={MODAL_ROW_STYLES} justify={'space-between'}>
+            <Col>
+                <Row gutter={[20, 0]}>
+                    <Col>
+                        <Button onClick={handlePrevStepButtonClick} disabled={step === 0} type={'text'}>
+                            ←
+                        </Button>
+                    </Col>
+                    <Col>
+                        <Button onClick={handleNextStepButtonClick} disabled={step === 2} type={'text'}>
+                            →
+                        </Button>
+                    </Col>
+                </Row>
+            </Col>
+            {
+                step === 2 && (
+                    <Button type={'sberDefaultGradient'} onClick={handleModalClose}>
+                        {OkMessage}
+                    </Button>
+                )
+            }
+        </Row>
+    )
+}
+
+type WelcomePopupImage = {
+    src: string
+    style: CSSProperties
+}
+
+type WelcomePopupStep = {
+    imageBackgroundColor: string
+    images: WelcomePopupImage[]
+    title: string
+    text: string
+}
+
+const IMAGE_COL_STYLES = { display: 'flex', alignItems: 'end' }
+const BODY_TEXT_STYLES = { fontSize: fontSizes.content, lineHeight: '24px', letterSpacing: '-0.01em' }
+
+export function WelcomePopup () {
+    const intl = useIntl()
+    const Step1Title = intl.formatMessage({ id: 'WelcomePopup.step1.title' })
```

Comment:
Looks like copy-paste. Better to store this in array and have only one dependency for memo
It will be easier to add  more steps later